docs/sources/dashboards/variables/add-template-variables/index.md

```diff
@@ -314,6 +314,10 @@ Currently only supported for Prometheus and Loki data sources. This variable rep
 
 Currently only supported for Prometheus data sources. The `$__rate_interval` variable is meant to be used in the rate function. Refer to [Prometheus query variables]({{< relref "../../../datasources/prometheus/template-variables#use-__rate_interval" >}}) for details.
 
+### $\_\_rate_interval_ms
+
+This variable is the `$__rate_interval` variable in milliseconds, not a time interval formatted string. For example, if the `$__rate_interval` is `20m` then the `$__rate_interval_ms` is `1200000`.
```

Comment:
```suggestion
This variable is the `$__rate_interval` variable in milliseconds, not a time-interval-formatted string. For example, if the `$__rate_interval` is `20m` then the `$__rate_interval_ms` is `1200000`.
```

Reply:
Is this supported by prometheus and Loki? Can we add that to the doc?

Reply:
I believe variables are all handled in Grafana and don't require specific support in the back ends. So, this should be handled by all of them.